cg/store/crud/read.py

```diff
@@ -313,25 +313,26 @@ def filter_cases_with_samples(self, case_ids: list[str]) -> list[str]:
 
     def get_cases_by_ticket_id(self, ticket_id: str) -> list[Case]:
```

Comment:
Would we also want to change the type of the ticket_id to int in this file?

Reply:
I see beneficial to have ticket id as int in all our codebase but that seems out of scope

Reply:
Yeah, out of scope but it should be possible once everything uses the Order table to get the ticket_id. I could try to see what would happen if the input is changed to int here though

Reply:
Seems to impact CLI commands and service instantiations. I will consider out of scope